app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -41,8 +41,11 @@ $govuk-cookie-banner-background-new: govuk-colour("white");
 }
 
 .gem-c-cookie-banner__buttons {
-  display: inline-block;
   vertical-align: middle;
+
+  @include govuk-media-query($from: desktop) {
+    display: inline-block;
```

Comment:
Why does this need to be declared here? I'm sure it's fine, I'm just curious.

Reply:
@andysellick It allows the buttons to display next to the cookie banner text on desktop.

## Without:
<img width="906" alt="Screen Shot 2019-05-30 at 15 03 27" src="https://user-images.githubusercontent.com/29889908/58638201-28ce1d00-82ec-11e9-98dc-b9e684a79b2e.png">

## With:
<img width="935" alt="Screen Shot 2019-05-30 at 15 03 32" src="https://user-images.githubusercontent.com/29889908/58638208-2cfa3a80-82ec-11e9-8087-dd269230184d.png">

Reply:
Oh, okay. I'm not sure I fully understand that still. Don't worry about it for now, but next time we meet up I'd be curious to see it in action.